tools/trustee/src/cli.rs

```diff
@@ -50,6 +83,9 @@ pub async fn cli_default() -> Result<(), Error> {
             let out = out.unwrap_or_else(|| trustee_home_dir.join("key"));
             trustee_keygen(&out).unwrap();
         }
+        Commands::Run { config_file } => {
+            trustee_run(config_file, &trustee_home_dir).await.unwrap();
```

Comment:
Do we want to go with `unwrap()` here (and other commands as well)?  With `unwrap()` I might get
```
thread 'main' panicked at tools/trustee/src/cli.rs:126:18:
called `Result::unwrap()` on an `Err` value: refusing to overwrite file: "/home/pvl/.trustee/https_key.pem"
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
whereas if I replace the `unwrap()` with just a quick fix like this:
```
.map_err(|e| Error::raw(clap::error::ErrorKind::InvalidValue, format!("{}\n", e.to_string())))?
```
the same error is reported as
```
error: refusing to overwrite file: "/home/pvl/.trustee/https_key.pem"
```
which seems much clearer.  Granted, both contain the important information but frankly I initially didn't even notice it in the unwrap message. :-)

Reply:
A broader question might be how we want to handle errors.  The fix I proposed above assumes we want to return `clap::Error` from `cli_default()` - but do we?  The only reason seems to be the `Cli::try_parse()?` call at the top of the function, command handlers apparently use `anyhow`.

Maybe we'd prefer to convert the `try_parse()` error to `anyhow` and then `cli_default()` returns anyhow errors, making it possible for command handlers to be called simply like
```
trustee_run(config_file, &trustee_home_dir).await?;
```
?